build-system/pr-check.js

```diff
@@ -430,6 +430,32 @@ function runYarnLockfileCheck() {
   }
 }
 
+/**
+ * Returns true if this is a PR build for a greenkeeper branch.
+ */
+function isGreenkeeperPrBuild() {
+  return (process.env.TRAVIS_EVENT_TYPE == 'pull_request') &&
```

Comment:
nit. no need for all these parens on the return

Reply:
I found the parens useful for readability and to make sure I didn't screw up the boolean logic. Happy to remove them if there's a general preference against parens.

Reply:
yeah i don't really care but on a first glance having the parens there makes me think "OH is there something im missing" (like grouping etc), since there is nowhere in the source code where we do this.